### PR TITLE
<Tabs/> - added size small

### PR DESCRIPTION
**Pull request by ChenAlmagor**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [x] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6083

File: src/Tabs/core/TabItem/index.js [hunk 1]
```diff
@@ -19,12 +19,14 @@ class TabItem extends React.Component {
...
-    const { item, onItemClick, isActive, dynamicProperties } = this.props;
+    const { item, onItemClick, isActive, dynamicProperties, size } = this.props;
 
     const containerProps = {
       key: item.id,
       onClick: () => onItemClick(item),
-      className: classNames(classes.tab, { [classes.active]: isActive }),
+      className: classNames(classes.tab, classes[size], {
+        [classes.active]: isActive,
+      }),
```
Comment (EgoziE) [inline src/Tabs/core/TabItem/index.js:29]: This contradicts the Stylable way :(

It should be like this:
```
      className: st(classes.tab, {
        size,
        active: isActive,
      }),
```

And the st.css should be:
```
.tab {
  -st-states: size(enum(small, medium)), active;
}

.tab:size(small) { ... }
.tab:size(medium) { ... }
```

I know it was already not the Stylable way regarding `active`, I'd appreciate it if you can change both ❤️

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/Tabs/core/TabItem/index.js (hunk 1 above)
Comment (ChenAlmagor) [reply src/Tabs/core/TabItem/index.js]: done

**Review by EgoziE — APPROVED** [review]

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@ChenAlmagor @EgoziE 

- [ ] Tab's height
Instead of 49px:
image
 It should be 42px:
image

**Review by EgoziE — APPROVED** [review]

**Pull request merged**